clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -114,6 +117,69 @@ def start_clipper(
             logger.warning("Error starting Clipper: {}".format(e.msg))
             raise e
 
+        def parse_configuration(config_path):
+            """ if a configuration file exists,
+            parse it and accordingly deploy models and applications
+
+            Parameters
+            ----------
+            config_path : str
+                Path to the configuration file.
+            """
+            if (config_path):
+                with open(config_path, "r") as stream:
+                    config = yaml.load(stream)
+                    if ("links" in config):
+                        for link in config["links"]:
+                            try:
+                                self.link_model_to_app(
+                                    config["links"][link]["app_name"],
+                                    config["links"][link]["model_name"])
+                            except ClipperException as e:
```

Comment:
This will only catch `ClipperException` objects thrown by `link_model_to_app`, but it won't catch `KeyError` exceptions in the case that fields are missing. We should also catch these exceptions and indicate to the user that required keys are missing by throwing a `ClipperException` with a message to the effect of "Required keys missing from Clipper link specification"

Reply:
Done